stepfunctions/src/main/scala/awscala/stepfunctions/StateMachine.scala

```diff
@@ -0,0 +1,46 @@
+package awscala.stepfunctions
+
+import awscala.DateTime
+import awscala.stepfunctions.ArnFormat.ResourceArn
+import com.amazonaws.services.stepfunctions.model._
+
+import scala.collection.JavaConverters._
+
+case class StateMachine(arn: String) {
+  def name: String = ArnFormat.parseArn(arn, ResourceArn)
```

Comment:
val?

Reply:
Good catch, I have this in some other places I'll update as well